### ISSUE TITLE
Invoice Processing for September

### ISSUE THREAD

**Issue by allancto** [rchain/bounties#1002]
### Benefit to RChain
September Trustmetric voted rewards properly processed. 

### Details
September Trustmetric voting closed on Oct 9, but we did not get invoices out until Oct 13. In addition to being very careful, we made some small improvements in the handoff process to Finance this month. In particular we automated out a manual step that led to last month’s problem. 

As of  today, Oct 16, we have all but 3 e-signs, and another one who has elected to donate their rewards back to the Cooperative and one in process. If you haven’t yet signed yet please do that asap. I’ve already reached out on Discord to everyone who hasn’t yet responded. Let me know if you have any questions. We will be handing off to Finance today all of the data we have, 

During the e-sign process several questions arose. For updating an eth_address we established a manual procedure for doing this by emailing the request for update which we will then confirm. There were several questions about the usability of the reward amount box in which the amount is currently written as a “hint” but must be typed over. Another question had to do with rewards which may be covered in the future by other payments (answer from Finance: please agree to current reward and deduct from any future actions). 

Looking forward, we have been in the process of proposing a Contributors Dashboard which will again improve usability and accuracy. In particular, updates to personal data such as eth_address will be easier to make, a one time agreement to Trustmetric voting will be sufficient instead of monthly agreement, and certain special cases such as donation or “hold” of voted rewards will be categorized and made routine. 

### Budget and Objective

_Estimated Budget of Task:_ **$[1200 (8 story points)]**
_Estimated Timeline Required to Complete the Task:_ **[Oct 9 - Oct 17]**
_How will we measure completion?_ **[Contributors notified and voted rewards properly transferred]**

### Legal

_Task Submitter shall not submit Tasks that will involve RHOC being transacted in any manner that (i) jeopardizes RHOC’s status as a software access token or other relevant and applicable description of the RHOC as an “asset”—not a security— or (2) violates, in any manner, applicable U.S. Securities laws._

**Comment by dckc** [comment]
story points? So this is development work? The measure of completion strongly suggests otherwise.

Where is the relevant code or other artifact?

**Comment by allancto** [comment]
@dckc yes they went out on Oct 13, and data handoff to Finance on Oct 17. Feedback from Finance is positive, they appreciate being able to see the entire scope of September invoices and the status of each. They've indicated transfers will happen next week. 

To answer your question what code is involved, you're correct most of the effort involved "test and support", examining all data to make sure everything is correct, and following up with everyone in DM to make sure all e-signs were responded to. Small mod to [e-sign code](https://github.com/whereyouatwimm/rchain-invoice). Discussion but no resolution yet around amount "hint", which was helpful for some but caused confusion for others. Most of the tweaks were on the admin side: added ability to process single invoice as well as batch; designed and hand produced (not implemented in code yet) new format for handoff so that Finance should be able to do transfers directly from a spreadsheet we produce. This saves one processing step and improves record keeping with transfer and invoices_summary in the same spreadsheet. The admin code hasn't been made public but can be shared on a 1-1 basis. Certain edges cases aren't addressed yet, particularly that of rewards voted but contributor not yet registered.

**Comment by dckc** [comment]
> The admin code hasn't been made public but can be shared on a 1-1 basis.

Right. Please do. Google apps knows me as dckc@madmode.com .
